design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -941,3 +984,151 @@ Libvirt configuration snippet:
     <rom enabled='no'/>
 </interface>
 ```
+
+## Appendix F: Concrete example for vDPA binding
+
+Present a concrete example of the actions needed at each integration
+point, considering the option suggested in the
+[design section](#design-details).
```

Comment:
leftover?

Reply:
What leftover do you see here?

Reply:
lines 990-992, it doesn't say anything, just describing what it should.